draft/ZEP0007.md

```diff
@@ -0,0 +1,371 @@
+---
+layout: default
+title: ZEP0007
+description: Strings
+parent: template
+nav_order: 1
```

Comment:
```suggestion
parent: draft ZEPs
nav_order: 7
```

Reply:
Hi @ivirshup. This is a minor change in the front matter so that ZEP is listed under the `draft` section, not the `template` section.